src/MLJMultivariateStatsInterface.jl

```diff
@@ -150,4 +150,1078 @@ metadata_pkg.(
+  requires a `Vector` containing class probabilities with probabilities specified using
+  the order given by `levels(y)` where y is the target vector.
+
+
+# Operations
+
+- `transform(mach, Xnew)`: Return a lower dimentional projection of the input `Xnew` having the same scitype as `X` above.
+- `predict(mach, Xnew)`: Return predictions of the target given
+  features `Xnew` having the same scitype as `X` above. Predictions
+  are probabilistic.
+- `predict_mode(mach, Xnew)`: Return the modes of the probabilistic predictions
+   returned above.
+
+
+# Fitted parameters
+
+The fields of `fitted_params(mach)` are:
+
+- `projected_class_means`: The matrix comprised of class-specific means as
+  columns (of size `(d,m)`), where d corresponds to input features and m corresponds to class.
+- `projection_matrix`: The matrix used to project `X` into a lower dimensional space.
+- `priors`: The estimated class priors.
+
+# Report
+
+The fields of `report(mach)` are:
+
+- `classes`: The classes seen during model fitting.
+- `out_dim`: The dimensions the model is projected to.
+- `class_means`: The matrix comprised of class-specific means as
+  columns (of size `(d,m)`), where d corresponds to input features and m corresponds to class.
+- `mean`: The mean vector.
+- `class_weights`: The weights of each class.
+- `Sb`: The between class scatter matrix.
+- `Sw`: The within class scatter matrix.
+- `nc`: The number of classes.
+
+# Examples
+
+```
+using MLJ
+
+BLDA = @load BayesianLDA pkg=MultivariateStats
+
+X, y = @load_iris
+
+model = BLDA()
+mach = machine(model, X, y) |> fit!
+
+Xproj = transform(mach, X)
+y_hat = predict(mach, x)
+labels = predict_mode(mach, x)
+```
+
+See also
+[`LDA`](@ref), [`SubspaceLDA`](@ref), [`BayesianSubspaceLDA`](@ref)
+"""
+BayesianLDA
+"""
+$(MMI.doc_header(SubspaceLDA))
+
+`SubspaceLDA`: Multiclass linear discriminant analysis. Suitable for high
+dimensional data (Avoids computing scatter matrices `Sw` ,`Sb`). The algorithm learns a
+projection matrix `P = W*L` that projects a feature matrix `Xtrain` onto a lower
+dimensional space of dimension `nc - 1` such that the trace of the transformed
+between-class scatter matrix(`Pᵀ*Sb*P`) is maximized relative to the trace of the
+transformed within-class scatter matrix (`Pᵀ*Sw*P`). The projection matrix is scaled
+such that `Pᵀ*Sw*P = mult*I` or `Pᵀ*Σw*P=mult/(n-nc)*I` (where `n` is the number of
+training samples, mult` is  one of `n` or `1` depending on whether `Sb` is normalized,
+`Σw` is the within-class covariance matrix, and `nc` is the number of unique classes
+in `y`) and also obeys `Wᵀ*Sb*p = λ*Wᵀ*Sw*p`, for every column `p` in `P`.
+Predicted class posterior probability for feature matrix `Xtest` are derived by
+applying a softmax transformation to a matrix `Pr`, such that  rowᵢ of `Pr` contains
+computed distances(based on a distance metric) in the transformed space of rowᵢ in
+`Xtest` to the centroid of each class.
```

Comment:
I'm not sure this doc string is correct. It doesn't seem to match the core pkg. In any case, I suggest we just duck out in this case with a referral to the core Pkg. I've reworked the classification part as per previous suggestion for vanilla LDA:

```suggestion
Multiclass subspace linear discriminant analysis (LDA) is a variation on 
ordinary LDA suitable for high dimensional data, as it avoids storing scatter 
matrices. For details, refer the 
[MultivariateStats.jl documentation](https://juliastats.org/MultivariateStats.jl/stable/). 

In addition to dimension reduction, probabilistic classification is provided. 
In the case of classification, the class probability for a new observation 
reflects the proximity of that observation to training observations 
associated with that class, and how far away the observation is from those 
associated with other classes. Specifically, the distances, in the transformed 
(projected) space, of a new observation, from the centroid of each target class, 
is computed; the resulting vector of distances (times minus one) is passed to a 
softmax function to obtain a class probability prediction. Here "distance" 
is computed using a user-specified distance function.
```